charter.md

```diff
@@ -0,0 +1,41 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
+
+## Scope
+
+The team works by prioritizing specific web platform features. The minimum requirements for a feature to be in scope are:
+
+- The feature is defined by a specification from the [IETF](https://www.ietf.org/), [Khronos Group](https://www.khronos.org/), [TC39](https://tc39.es/), [W3C](https://www.w3.org/), or [WHATWG](https://whatwg.org/). Other specifications can be considered on a case-by-case basis.
+- The feature can be tested by automated tests in [web-platform-tests](https://github.com/web-platform-tests/wpt). Manual tests and other test suites can be considered on a case-by-case basis.
+
+Browser features that don't affect web developers, like bookmarks, are out of scope.
+
+Writing specifications is not in scope, but the team can liaise with and work within other working groups to make progress in key areas.
+
```

Comment:
Not sure if this should be in "scope" or somewhere else, but possibly something like the following:

```
Initially the group will produce one set of interop metrics per year. This may be changed in the future by consensus decision.

If at any time the group is unable to reach consensus which features to include in its metrics, or the set of features which has consensus is considered insufficient to fulfill the goals of the group, the effort will be paused until the situation can be resolved.
```

(the last bit perhaps doesn't say anything new, but I want it to be clear that if e.g. we only get consensus on one feature we don't need to go ahead with producing an Interop metric from that, but can instead just say "this isn't working, we need to try something different")

Reply:
I like the wording, and for lack of a better place put it at the end of the scope section.

Reply:
Given that we are in april 23, it seems weird to not go ahead and add 2023?

Reply:
Do you mean in the paragraph that starts "The team is responsible for maintaining"? I'll update it.